x/interchainstaking/keeper/proposal_handler.go

```diff
@@ -210,7 +217,12 @@ func (k *Keeper) HandleUpdateZoneProposal(ctx sdk.Context, p *types.UpdateZonePr
 				return err
 			}
 			zone.Is_118 = boolValue
-
+		case "transfer_channel":
+			_, found = k.IBCKeeper.ChannelKeeper.GetChannel(ctx, transfertypes.PortID, change.Value)
+			if !found {
+				return errors.New("unable to fetch channel")
+			}
+			zone.TransferChannel = change.Value
```

Comment:
I'm not sure about this case; once qAssets have left Quicksilver, we cannot change the channel. Are we able to check the balance of the escrow account for this channel?